fitbenchmarking/results_processing/base_table.py

```diff
@@ -543,15 +551,21 @@ def vals_to_colour(vals, cmap, cmap_range, log_ulim):
         """
         log_vals = np.log10(vals)
         log_llim = min(log_vals)
-        norm_vals = (log_vals - log_llim) /\
+        numerator_norm_vals = np.array([np.nan if
+                                        (np.isinf(val) and
+                                         np.isinf(log_llim))
+                                        else val - log_llim
+                                        for val in log_vals])
+        norm_vals = numerator_norm_vals /\
             (log_ulim - log_llim)
         norm_vals[norm_vals > 1] = 1  # applying upper cutoff
-        norm_vals[np.isnan(norm_vals)] = 1  # deal with nans
         # trimming colour map according to default/user input
         norm_vals = cmap_range[0] + \
             norm_vals*(cmap_range[1] - cmap_range[0])
         rgba = cmap(norm_vals)
-        hex_strs = [mpl.colors.rgb2hex(colour) for colour in rgba]
+        hex_strs = ["#e5e4e2" if np.isinf(v) else
```

Comment:
Is this a CSS colour? We could use the matplotlib colors if it is:
```
mpl.colors.to_hex("whitesmoke")
```

Reply:
I got the colour from this [link](https://htmlcolorcodes.com/colors/shades-of-gray/). It was `platinum`. But `whitesmoke` looks equally good.